.github/workflows/full_stack_tests.yml

```diff
@@ -189,105 +189,136 @@ jobs:
               break
             fi
           done
+
       - name: Checkout repository so that local actions can be used
         uses: actions/checkout@v4
+
       - name: Merge develop and set up dependencies
         uses: ./.github/actions/merge-develop-and-set-up-dependencies
+
       - name: Generate build files
         uses: ./.github/actions/generate-build-files
+
       - name: Generate modified suite name for artifacts
         id: generate_suite_name_for_artifacts
         # Replace slashes in the filename with underscores.
         run: |
           SUITE_NAME=${{ matrix.suite.name }}
           echo "MODIFIED_SUITE_NAME=${SUITE_NAME//\//_}" >> $GITHUB_OUTPUT
+
+      # Desktop Test (Attempt 1)
       - name: Run Desktop Acceptance Test ${{ matrix.suite.name }}
+        id: desktop-test-attempt-1
         # TODO(#22448): Remove once all suites are fixed.
         if: ${{ steps.check_skip.outputs.SKIP_SUITE != 'true' }}
         run: >
           set -o pipefail;
-          VIDEO_RECORDING_IS_ENABLED=1
```

Comment:
Why is this being dropped? Sometimes a test fails on the first attempt and not on the second attempt; in such cases it would still be helpful to have video recordings of the first attempt.

Reply:
It would be helpful. However, this was added mistakenly, at initial implementation, you said not to record first time. So, I dropped it. However, if you think we should keep it, I can revert it.

Reply:
Ah yes, thanks -- I think we have a clearer understanding of the requirements now anyway. I would suggest adding it, yes.